browser/admin/src/integrator/AdminIntegratorSettings.ts

```diff
@@ -109,7 +112,9 @@ class SettingIframe {
 				uploadPath: this.PATH.autoTextUpload(),
 			},
 			{
-				sectionTitle: 'Wordbook',
+				sectionTitle: 'Custom Dictionaries',
```

Comment:
Not blocker: perhaps we should change all error modal messages from `wordbook` to `Custom Dictionaries`.